frontend/public/components/utils/status-box.tsx

```diff
@@ -134,17 +133,6 @@ export const StatusBox: React.FC<StatusBoxProps> = (props) => {
       return <AccessDenied message={loadError.message} />;
     }
 
-    if (loaded && loadError instanceof TimeoutError) {
```

Comment:
Is there a way to detect a browser timeout when using `fetch`?

I suspect this is here to warn if showing really old data from Redux.

Reply:
I guess it would just fall back to showing the error page now, but we're losing some functionality. (You no longer get any data, even if stale.)

Reply:
I couldn't find a way to know if browser timed out using fetch. The implementation we had was the workaround for using fetch with timeouts.

Reply:
Not sure how we'll reach a state of showing stale data while doing a fetch request. I would assume if we're watching the resource for changes then we would be using `Firehose` or `useK8sWatchResource` hook which doesn't use `fetch`.

Or if using fetch to do some kind of polling, it should be the parent component's responsibility to handle failures to API requests properly.

Reply:
> Not sure how we'll reach a state of showing stale data while doing a fetch request. I would assume if we're watching the resource for changes then we would be using `Firehose` or `useK8sWatchResource` hook which doesn't use `fetch`.

But it was already working. Firehose uses fetch under the hood to list resources before watching. I think we were falling back to the data in Redux on timeouts, which was the reason for the timeout race. Sort of a poor man's offline mode.

Should we keep the timeout and make it bigger or make it optional for requests we know are long-running like Helm?

Reply:
Maybe, my first idea was to actually expose the timeout parameter to `coFetchJSON` as well so that API requests like Helm install/upgrade can specify a bigger timeout. Or we can simply increase the default timeout. WDYT @christianvogt?

Reply:
Never bothered with this case...
I wonder what kind of error is thrown from the browser when it times out.

Quite frankly we are better off making a specific timeout in firehose that notifies the child component if the request is taking long and we want to show such a message. At least we continue waiting on the request and don't bail and error out.

Reply:
@rohitkrai03 use a 60 second timeout and remove the timeout for the helm install call.

Reply:
Updated the PR as per the discussion.